src/stories/index.js

```diff
@@ -40,7 +33,31 @@ storiesOf("NavbarTop", module)
   .add("NavbarTopNotif", () => <NavbarTopNotif />)
   .add("NavbarTopProfile", () => <NavbarTopProfile />)
 
-storiesOf("NavbarTopNotif", module).add("NavbarTopNotifBubble", () => (
-  <NavbarTopNotifBubble />
+storiesOf("NavbarTopNotif", module).add("NavbarTopNotif0", () => (
```

Comment:
instead of Bubble0 use more semantic names  like propsWithOnotif, propsWith1notif etc...
then if you use the same name everywhere you can do something awesome like ```<MyComponent {...propsWithOnotif} />```